exchanges/deribit/deribit_types.go

```diff
@@ -529,6 +530,45 @@ type PrivateTradeData struct {
 	Order  OrderData   `json:"order"`
 }
 
+// CancelResp represents the detail of canceled order.
+type CancelResp struct {
+	InstrumentName string `json:"instrument_name"`
+	Currency       string `json:"currency"`
+	Result         []struct {
```

Comment:
The only thing I could suggest is to make this struct `[]PrivateCancelData` instead so as to only maintain one set of data